### PR TITLE
Renamed .selector to .selector-item as it is duplicated

### PR DESCRIPTION
**Pull request by kmturley**
Suggestion to use .selector as top level class only, and use
.selector-item for duplicates items below

Fix for this issue:
https://github.com/indrimuska/angular-selector/issues/14

### CODE REVIEW

**Comment by indrimuska** [comment]
Hey @kmturley, thanks for sharing this idea. It looks good to me, just perform the following updates and let's create a new Major version (since it will be incompatible with the current one) :+1:

File: src/angular-selector.css [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 /* Selector */
 [selector] { display: none; position: relative; }
-[selector].selector { display: block; }
-.selector { font-size: 1em; line-height: normal; color: #495c68; text-shadow: 0px 1px 0px rgba(255, 255, 255, .5); }
+[selector].selector-item { display: block; }
```
Comment (indrimuska) [inline src/angular-selector.css]: Change all `selector-item` class to `selector-container`, we will add this class only to the main container.

File: src/angular-selector.css [hunk 2]
```diff
@@ -5,8 +5,8 @@
...
+.selector-item { font-size: 1em; line-height: normal; color: #495c68; text-shadow: 0px 1px 0px rgba(255, 255, 255, .5); }
```
Comment (indrimuska) [inline src/angular-selector.css]: In order to use just `.selector-container` class, this should be:

``` css
.selector-container,
.selector-container input { font-size: ... }
```

File: src/angular-selector.css (hunk 1 above)
Comment (indrimuska) [inline src/angular-selector.css]: Also, in order to see the selector only after initialization, we have to support attribute and class restriction in the css:

``` css
[selector],
.selector { display: none; }
.selector-container { display: block; position: relative; }
```

File: src/angular-selector.js [hunk 3]
```diff
@@ -459,7 +459,7 @@
 		.module('selector', [])
 		.run(['$templateCache', function ($templateCache) {
 			$templateCache.put('selector/selector.html',
-				'<div class="selector" ng-attr-dir="{{rtl ? \'rtl\' : \'ltr\'}}" ' +
+				'<div class="selector-item" ng-attr-dir="{{rtl ? \'rtl\' : \'ltr\'}}" ' +
```
Comment (indrimuska) [inline src/angular-selector.js]: Update class to `selector-container`.

File: src/angular-selector.js [hunk 4]
```diff
@@ -473,7 +473,7 @@
 								'</div>' +
 							'</li>' +
 						'</ul>' +
-						'<input class="selector" ng-model="search" placeholder="{{!hasValue() ? placeholder : \'\'}}" ng-disabled="disabled">' +
+						'<input class="selector-item" ng-model="search" placeholder="{{!hasValue() ? placeholder : \'\'}}" ng-disabled="disabled">' +
```
Comment (indrimuska) [inline src/angular-selector.js]: Remove class attribute.

**Comment by kmturley** [comment]
@indrimuska ok have made those changes! seems to work fine for me. This will make it alot easier to control the top level classes now :)

**Comment by indrimuska** [comment]
Ok, you only missed the directive restriction "C", but don't worry I will add it for you.
Thank you man! :)

**Comment by indrimuska** [comment]
This new feature has been released in version [1.0.0](https://github.com/indrimuska/angular-selector/releases/tag/1.0.0).
This also closes #14.

**Pull request merged**